### ISSUE TITLE
Date picker not aligned when there is fixed top menu on page in bootstrap.

### ISSUE THREAD

**Issue by lonerunner** [eternicode/bootstrap-datepicker#1584]
When there is a fixed top menu on page in bootstrap we have to add some element with margin top or to add margin to body.

In that case the datepicker is out of input element basically it is shown over input element because it calculates element style top in my case 50px less.

So for example if i open datepicker and inspect with div with datepicker class i see 

```
element.style {
    display: block;
    left: 1261.5px;
    top: 176.8px;
}
```

![screenshot - 180915 - 22 13 53](https://cloud.githubusercontent.com/assets/1761871/9970022/b3b4a88e-5e52-11e5-9c3b-59b1d0157ffb.png)

If i edit that and add 50px, the datepicker shows in place. I can add margin-top: 50px to datepicker class and it would work but that is just quick fix for my specific case.

![screenshot - 180915 - 22 14 40](https://cloud.githubusercontent.com/assets/1761871/9970100/53e62256-5e53-11e5-92d6-b5a36562c55d.png)

**Comment by hebbet** [comment]
what version are you using?

**Comment by lonerunner** [comment]
Bootstrap v3.3.4
With Bootflat UI
And bootstrap-datepicker3.standalone

I use standalone because Bootflat UI is interfering with look of dropdown menu. In Bootflat dropdown menu is dark grey so numbers are not visible. 
But nevertheless even if i disable Bootflat UI the dropdown is out of align when fixed menu is on page.

**Comment by hebbet** [comment]
what version of bootstrap-datepicker are you using?

**Comment by lonerunner** [comment]
I updated it yesterday so i use the latest version
Datepicker for Bootstrap v1.5.0-dev

**Comment by hebbet** [comment]
hm, dist-files might be outdated. did you build from source?

a jsfiddle could be helpful

**Comment by lonerunner** [comment]
I used dist files.
Here is a fiddle example https://jsfiddle.net/vddcse04/1/
It's not the best aligned but it show the problem, if you don't have margin top or bottom on body or first element when using fixed menu, the first element will overlap and go behind the menu. 
But that makes the problem on datepicker as i said.
Try removing css in example and you can see that datepicker is aligned perfectly but than jumbotron is hidden behind the menu.

**Comment by gnuchu** [comment]
+1

**Comment by raefa** [comment]
Not a solution but a workaround:

**Comment by raefa** [comment]
This is not a solution ... rather a work around. Calculate the height of the menu at the top and add it to the datepicker on "show". 

    $(".date input").datepicker().on("show", function(e) {
    var top = $("#organization-header").height() + parseInt($(".datepicker").css("top"));
      $(".datepicker").css("top", top);
    });

**Comment by fernandowalla** [comment]
Instead of margin-top/margin-bottom use:

body {
	font-family: $open_sans;
	font-size: $font_base;
	color: $font_color;
	padding-top: $navbar_height_int;
	padding-bottom: $footer_height;
}

**Comment by PeterSawyer** [comment]
Yep @fernandowalla is correct, for some reason my body had margin-top instead of padding-top for my fixed nav.

Fixed with this: -

`body {
	padding-top: 60px;
}`
